go/vt/topo/k8stopo/file.go

```diff
@@ -42,11 +45,54 @@ type NodeReference struct {
 	value string
 }
 
+func packValue(value []byte) ([]byte, error) {
+	var gzipped bytes.Buffer
+
+	zw := gzip.NewWriter(&gzipped)
+	_, err := zw.Write(value)
+	if err != nil {
+		return []byte{}, fmt.Errorf("gzip write error: %s", err)
+	}
+
+	err = zw.Close()
+	if err != nil {
+		return []byte{}, fmt.Errorf("gzip close error: %s", err)
+	}
+
+	encoded := base64.StdEncoding.EncodeToString(gzipped.Bytes())
```

Comment:
It probably won't make that much of a difference, but if you chain your gzip writer using `https://golang.org/pkg/encoding/base64/#NewEncoder`, it should prevent some extra allocations. Same thing on the unpack with the Decoder.

Reply:
Good idea! I'll update it

Reply:
@derekperkins updated